migrating_from_ocp_3_to_4/planning-migration-3-4.adoc

```diff
@@ -135,6 +135,46 @@ If your {product-title} 3.11 cluster used the `ovs-subnet` or `ovs-multitenant`
 
 For more information, see xref:../networking/network_policy/about-network-policy.adoc#about-network-policy[About network policy].
 
+[id="migration-network-traffic-migration-strategies"]
```

Comment:
The information in your section does not work in this assembly because it contains far more information than the other sections. The subsections of this assembly are similar to reference modules. Your subsection is closer to a procedure. Also, the other sections are "considerations" based on the differences between OCP 3 and 4. Your section is a description of different strategies, with no explanation of why one strategy might be more appropriate than another.